Autoaid/src/Components/CTA.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+function CTA() {
+  return (
+    <>
+      <div className="flex flex-col md:flex-row relative">
+        <div className="cta-car z-40 absolute left-0">
+          <img
+            src="/service-Images/cta-car.png"
```

Comment:
This is a public component and its Images should be seperate from the service image folder
I have finished it from my end but haven't push it.